xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -123,9 +122,9 @@ void CPVRRecordings::GetSubDirectories(const std::string &strBase, CFileItemList
 
     std::string strFilePath;
     if(strUseBase.empty())
-      strFilePath = StringUtils::Format("pvr://recordings/%s/%s/", strBasePath.c_str(), strCurrent.c_str());
+      strFilePath = StringUtils::Format("pvr://" PVR_RECORDING_BASE_PATH "/%s/%s/", current->IsDeleted() ? PVR_RECORDING_DELETED_PATH : PVR_RECORDING_ACTIVE_PATH, strCurrent.c_str());
     else
-      strFilePath = StringUtils::Format("pvr://recordings/%s/%s/%s/", strBasePath.c_str(), strUseBase.c_str(), strCurrent.c_str());
+      strFilePath = StringUtils::Format("pvr://" PVR_RECORDING_BASE_PATH "/%s/%s/%s/", current->IsDeleted() ? PVR_RECORDING_DELETED_PATH : PVR_RECORDING_ACTIVE_PATH, strUseBase.c_str(), strCurrent.c_str());
```

Comment:
please skip each deleted item and add a comment that we do that as we never call this method in case we show deleted items.